src/Controller/EnMarche/MembershipController.php

```diff
@@ -65,12 +66,56 @@ public function registerAction(Request $request, GeoCoder $geoCoder, Authorizati
     /**
      * This action enables a guest user to adhere to the community.
      *
+     * @Route("/adhesion", name="app_membership_register")
```

Comment:
Now we have two route with the same exact pattern : `/adhesion`. So the router is gonna stop on 1st one he found. IMO, we should just keep the old route name and that's all.